recipes/deco/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class DecoConan(ConanFile):
+    name = "deco"
+    license = "Apache-2.0 WITH LLVM-exception"
```

Comment:
```suggestion
    license = "Apache-2.0-WITH-LLVM-exception"
```

white space are not allowed